README.md

```diff
@@ -0,0 +1,150 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Hello World! <a name="about-project"></a>
+
+**Hello World!**
+Now that you're familiar with Rails and React, it's time to put them together in a new kind of 'Hello World!' app. This exercise is going to have you create a React front-end with a Rails back-end and connect them to display a random message.
+
+## 🛠 Built With <a name="built-with"></a>
+
+Rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+</details>
+
+<details>
+  <summary>Server</summary>
+</details>
+
+<details>
+<summary>Database</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Create a new comments**
+- **Get it display on the UI**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Command line or terminal with Rails installed.
+- IDE such as VsCode.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  https://github.com/Alabi12/hello-react-front-end.git
+```
+```
+  Rails backend
+
+   https://github.com/Alabi12/hello-rails-back-end.git
+```
+
+## Link to Hello_React_Front_End (http://127.0.0.1:5000/api/greetings)
```

Comment:
- [ ] [OPTIONAL] Kindly remove this url as you have already indicated the correct URL to the Front-end design in the backend README.